AdobeStockAsset/Model/Config.php

```diff
@@ -12,7 +12,7 @@
 use Magento\AdobeStockAssetApi\Api\ConfigInterface;
 
 /**
- * Class Config
+ * Used for managing an Adobe stock module config functionality
```

Comment:
Do you think it will make sense to move this class to AdobeStockImageAdminUI module? How is it used?
```suggestion
 * General cofiguration of Adobe Stock Assets functionality
```

Reply:
I think that this is an excellent idea. This class identifies is integration enabled or not. The asset module looks more like a solid structure which represents an Adobe Stock object. So to have this config class outside this module is the right choice.

Can we also rename it from the `Config` to the `IsAdobeStockIntegrationEnabled`? The class name `config` is an old and even ancient decision for naming config related helper classes.

Reply:
I think we can do this in the later in the 1.1-develop. Am I right?